x-pack/plugins/security_solution/public/entity_analytics/components/risk_score_enable_section.tsx

```diff
@@ -177,7 +136,59 @@ const RiskScoreUpdateModal = ({
   );
 };
 
-export const RiskScoreEnableSection = () => {
+const RiskEngineHealth: React.FC<{ currentRiskEngineStatus?: RiskEngineStatus | null }> = ({
+  currentRiskEngineStatus,
+}) => {
+  if (!currentRiskEngineStatus) {
+    return <EuiHealth color="subdued">{'-'}</EuiHealth>;
```

Comment:
show "-" if we dont know the risk engine status

Reply:
**nitpicking:** We use a special character in many places to represent the absence of data. You could use it by calling `getEmptyValue()`.